lib/vmdb/appliance.rb

```diff
@@ -107,70 +107,73 @@ def self.log_diagnostics
       end
     end
 
-    private
-
-    def self.get_build
-      build_file = File.join(File.expand_path(Rails.root), "BUILD")
+    # start of private class methods
+    class << self
```

Comment:
Good, I can't decide which I like better, private_class_method or singleton class syntax.  If you both agree, that helps!